tests/test_client_document.py

```diff
@@ -0,0 +1,32 @@
+import responses
+import json
+
+from .helpers import mock_file, ClientTestCase
+
+
+class TestClientDocument(ClientTestCase):
+
+    def setUp(self):
+        super(TestClientDocument, self).setUp()
+        self.base_url = '{}/documents'.format(self.base_url)
```

Comment:
can we use string formater `f'{self.base_url}/documents`. Do the same across the PR